### ISSUE TITLE
[RA] Nerf Demo Truck and A-Bomb damage against air units.

### ISSUE THREAD

**Issue by SoScared** [OpenRA/OpenRA#10799]
At ground 0 everything ought to be killed as the current state but IMO for air units, the range is too wide (aproximately 10 cells radius).

Hinds:
![hinds 1](https://cloud.githubusercontent.com/assets/2715583/13203028/56358b6c-d8ad-11e5-80e6-166699841d15.png)
![hinds 2](https://cloud.githubusercontent.com/assets/2715583/13203029/579987d8-d8ad-11e5-874c-e0f973fec5d7.png)

Yaks:
![yak 1](https://cloud.githubusercontent.com/assets/2715583/13203035/8d4340ae-d8ad-11e5-8324-faa4ce889d4f.png)
![yak 2](https://cloud.githubusercontent.com/assets/2715583/13203037/91fe8b3a-d8ad-11e5-8a39-a2b5b8f7d7da.png)

Logically air units are more resilient against nuke blasts than ground targets.

Balance wise I think it's a good idea to not allow the nuke truck to completely wipe out a nearby (very expensive) air fleet 7-10 cells away. Scouting for nuke trucks should be rewarded. Sniping nuke trucks with air units already guarantees the attacker a loss ($1350 Hind/YAK).

**Comment by fgdzfggsfgsfgsf** [comment]
I agree, it feels a little large for killing infantry as well but just this is an improvement.

**Comment by MustaphaTR** [comment]
Infantry is OK i think, but air should be nerfed.

**Comment by kyrylo** [comment]
I agree, it's a bit large. I would make it smaller only by 1, though. Air units are mobile enough to escape from demo trucks.

**Comment by klaas1** [comment]
Also, the demo truck is one of the most fun units in the game, and pretty risky to use. I don't mind it being high risk/ high reward. Please don't nerf it to the point people stop using them, because the game needs to be fun, not only perfectly balanced.

**Comment by Micr0Bit** [comment]
i agree with @klaas1

**Comment by SoScared** [comment]
@kyrylo Demo trucks can't be expected to be anticipated well tho - with exceptions they're spotted seconds before it's detonation and players in general don't really have the action points to move off their air units. I'd think a 7 cell guarantee-kill radius would be a good compromise.

@klaas1 I'm arguing the point towards air units exclusively. Having a 10 cell radius guarantee-kill for aircraft looks unreasonable. I don't think the fun aspect would be affected much given that aircrafts are never considered targets for the demo truck despite it's effect against them.

**Comment by Micr0Bit** [comment]
@SoScared : what makes you think that aircrafts are(were) never considered targets for a demo ?

the reason for our nuclear arsenal being so effective against yaks , is because they have the lowest health of all aircrafts ... 
the repulsion "may" or "may" not have a negative effect on idling planes , as they get more spread out

but as for all other aircraft types .... you have a shitload of space to hide them... and all of them have way more sight-radius then a demo-truck ... 

hell ... aircraft is the the #1 demotruck-killer ... as you just have to be "aware" of your surroundings

**Comment by MustaphaTR** [comment]
I think we need to decrase their damage against `Light` armor. Most of light armored units have too low HP that easily killed by Nuke Warhead. `MiniNuke` (also `Atomic:`) currently isn't set any Warhead vs Armor percent that makes all 100% as default.

**Comment by SoScared** [comment]
@Micr0Bit Demo Trucks aren't built by the player with killing off air units in mind. But that's a minor point. In the heat of battle players just wont micro away their air units with ease. Once you spot a Demo Truck on your doorstep (or on the field) you generally don't have the time to move off your air fleet as your main objective is to stop the truck as soon as humanly possible.

 A few cells nerf of the guarantee-kill range seems reasonable to me.

**Comment by StexoO** [comment]
Hey, the Demolition Truck transports a __Nuclear Bomb__.
https://www.youtube.com/watch?v=qjnm3V0xYjI
Maybe its price could be increased instead of having light tanks or planes surviving nuclear bombs ? :yum:

**Comment by SoScared** [comment]
I just realized I made a horrible mistake with the issue title. It's meant to say strictly "Nerf Demo Truck's A-Bomb damage against air units", not "Nerf Demo Truck **and** A-Bomb damage against air units. 

I never meant to say the Demo Truck should be nerfed in general but that's what the discussion coagulates around. Sorry for wasting your time guys:/

I'll be making a new issue at a later time. If someone could close this issue that'd be great.
